typings/docker-toolbelt/index.d.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ * Copyright 2020 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare module 'docker-toolbelt' {
```

Comment:
Looks like it's declared as a class there though, so might need some changes/experimentation to get it to work.

Reply:
I've tried that this morning, but haven't been able to get it working.  I wonder actually if these type declaration should be in `balena-io-modules/docker-toolbelt`, because the implementations are not defined there (they are from `dockerode`, which `docker-toolbelt` extends).

Reply:
It should definitely work properly (it does in the supervisor) because of the extends here: https://github.com/balena-io-modules/docker-toolbelt/blob/master/lib/docker-toolbelt.d.ts#L11

I'd check versions etc. If you still can't get it to work I'd much rather we reuse the dockerode types, than bring in our own.